packages/jest-expo/src/snapshot/createPlatformResolver.js

```diff
@@ -1,26 +1,29 @@
 const path = require('path');
 
-const testDirectory = `__tests__`;
```

Comment:
probably doesn't need to be deleted

Reply:
what do you mean? should I leave the unused var?